API/Pagination.md

```diff
@@ -8,7 +8,7 @@ There are different ways to enable pagination and to paginate resources, however
 
 ## Enabling pagination
 
-In most cases, pagination is enabled through query string. Query string lets clients pass additional parameters which are not appropriate to send in headers or the body. A URL with pagination applied might look like this: `/api/v1/countries?page=5`. In the example, we're fetching countries and requesting the 5th page of results. Query string is the preferred way to enable pagination and also the one we use.
+In most cases, pagination is enabled through query string. Query string lets clients pass additional parameters which are not appropriate to send in headers or the body. A URL with pagination applied might look like this: `/api/v1/countries?page=5`. In the example, we're fetching countries and requesting the 5th page of results. Query string is the preferred way to enable pagination, but you can keep it also enabled by default, where a request without pagination query strings is handled as `page=1`.
```

Comment:
I'd say `you should keep it also enabled by default`, you can sounds like it's nice to have.